src/frequenz/sdk/timeseries/_quantities.py

```diff
@@ -190,7 +190,9 @@ def __add__(self, other: Self) -> Self:
         """
         if not type(other) is type(self):
             return NotImplemented
-        return type(self)(self._base_value + other._base_value)
+        summe = type(self).__new__(type(self))
```

Comment:
Did you asked @matthias-wende-frequenz-GTP to write this code? :de:

Reply:
Is a built-in, not a keyword, and it is probably `pylint` being anal, it doesn't like people redefining built-ins :+1: